docker/Dockerfile.oidcprovider

```diff
@@ -1,12 +1,4 @@
-# Derived from the "testprovider" container from
-# https://github.com/mozilla-parsys/docker-test-mozilla-django-oidc.
-# Only the redirect_urls specified in "fixtures.json" are being
-# modified to fit the needs of the docker setup.
-
-FROM mozillaparsys/oidc_testprovider
+FROM mozilla/oidc-testprovider:oidc_testprovider-v0.9.3
```

Comment:
Existing tags are here:

https://hub.docker.com/r/mozilla/oidc-testprovider/tags?page=1&name=testprovider

We want a specific tag and not `latest` because it's nice to intentionally upgrade rather than accidentally upgrade.